cmd/build/test.go

```diff
@@ -143,15 +143,19 @@ func (t Test) Integration(ctx context.Context, self run.DependencyIDer, filter s
 		env["PKO_TEST_LATEST_BOOTSTRAP_JOB"] = url
 	}
 
+	coverPkg := dotFilepathJoin("...") + "," +
+		dotFilepathJoin("apis", "...") + "," +
+		dotFilepathJoin("pkg", "...")
+
 	tArgs := []string{
 		"go", "test",
-		"-tags=integration", "-coverprofile=.cache/integration/cover.txt",
+		"-tags=integration", "-coverprofile=" + filepath.Join(".cache", "integration", "cover.txt"),
 		f, "-race", "-test.v", "-failfast", "-timeout=20m", "-count=1", "-json",
-		"-coverpkg=./...,./apis/...,./pkg/...", "./integration/...", "|", "gotestfmt", "--hide=empty-packages",
+		"-coverpkg=" + coverPkg, dotFilepathJoin("integration", "..."), "|", "gotestfmt", "--hide=empty-packages",
```

Comment:
Not a file path, only use forward slashes

Reply:
Fixed